package.json

```diff
@@ -4,7 +4,7 @@
   "description": "JSON API-Compliant Serialization for your ORM",
   "main": "es5/src/mapper.js",
   "scripts": {
-    "prepublish": "tsd install && tsc",
+    "install": "tsd install && tsc",
```

Comment:
Good catch on not providing the setup after an `npm install` (I actually forgot about it), however, deleting the prepublish script will disrupt safe npm publish. Could you leave the `prepublish` and `post-install` to both install typings and compile typescript?